src/gui/folderman.cpp

```diff
@@ -766,10 +757,54 @@ void FolderMan::slotServerVersionChanged(Account *account)
     }
 }
 
-void FolderMan::slotScheduleFolderOwningFile(const QString& path)
+void FolderMan::slotWatchedFileUnlocked(const QString& path)
 {
     if (Folder* f = folderForPath(path)) {
-        slotScheduleSync(f);
+        f->scheduleThisFolderSoon();
+    }
+}
+
+void FolderMan::slotScheduleFolderByTime()
+{
+    foreach (auto& f, _folderMap) {
+        // Never schedule if syncing is disabled or when we're currently
+        // querying the server for etags
+        if (!f->canSync() || f->etagJob()) {
```

Comment:
Discard this comment, when i wrote it i tought the other timer was runing every 5 minutes.